js-api-doc/value/color.d.ts

```diff
@@ -1,5 +1,83 @@
+import {List} from 'immutable';
+
 import {Value} from './index';
 
+/** The HSL color space name. */
+export type ColorSpaceHSL = 'hsl';
```

Comment:
Sorry I missed this in the proposal review, but in the Sass API we follow the Dart convention of title-casing acronyms longer than two letters in camel-case identifiers (so this should be `ColorSpaceHsl`).